dapp/src/components/SignTransferAuth.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react'
+import { useWeb3React } from '@web3-react/core'
+import { useStoreState } from 'pullstate'
+import { fbt } from 'fbt-runtime'
+
+import { ethers } from 'ethers'
+
+import ContractStore from 'stores/ContractStore'
+import GetOUSD from 'components/GetOUSD'
+
+const SignTransferAuth = ({}) => {
+  const { account, active, library } = useWeb3React()
+  const [dstAddress, setDstAddress] = useState('')
+  const [sig, setSig] = useState(null)
+  const [error, setError] = useState(null)
+  const { ognStaking } = useStoreState(ContractStore, (s) => {
+    if (s.contracts) {
+      return s.contracts
+    }
+    return {}
+  })
+
+  return (
+    <>
+      <div>
+        <div className="content-holder flex-grow d-flex flex-column shadow-div">
+          {active && (
+            <div>
+              {' '}
+              on {ognStaking.address} Transfer stakes from {account} to:
```

Comment:
Would be cool to use `fbt` so this text is translateable

Reply:
This is probably going to be only used once or twice a quarter. Fine not to translate it, or do more work than making sure it works.

Reply:
Yeah, this doesn't need translations or to look pretty at all. Let's save Aure's time for more important stuff.

Reply:
Makes sense, got it